classes/class-wc-connect-stripe.php

```diff
@@ -63,43 +63,6 @@ public function get_oauth_url( $return_url = '' ) {
 			return $result->oauthUrl;
 		}
 
-		public function create_account( $email, $country ) {
-			$response = $this->api->create_stripe_account( $email, $country );
-			if ( is_wp_error( $response ) ) {
-				return $response;
-			}
-			return $this->save_stripe_keys( $response );
-		}
```

Comment:
Hi @chickenn00dle, 

Our `init_core_wizard_payments_config` rely on this `create_account` function https://github.com/Automattic/woocommerce-services/blob/develop/woocommerce-services.php#L623. Do we need to make change to how `init_core_wizard_payments_config` works?

Reply:
Good catch!

It looks like this functionality becomes redundant once deferred account creation is removed.

From what I am seeing, `$user_elected_to_create_stripe_account` will always be false here:
https://github.com/Automattic/woocommerce-services/blob/bca5be594c939770c5d377ae9cbcd6c47110256d/woocommerce-services.php#L596

And all of the conditional blocks in `init_core_wizard_payments_config` are ignored.

I tested this by removing the function and the line where it is called and the Stripe onboarding flow worked with no issues:
https://github.com/Automattic/woocommerce-services/pull/2008/commits/079ce1cb72501eda4f569faf3a782e92f8d886c1

I may be missing something though (is this used in any other scenario?) as I'm not very familiar with WC Services, so it'd be great if you could confirm the above.